include/Action.hpp

```diff
@@ -0,0 +1,10 @@
+// Copyright 2019 Jack Khuu, Justin Johnson, Lukas Joswiak
+
+#ifndef INCLUDE_ACTION_HPP_
+#define INCLUDE_ACTION_HPP_
+
+namespace Action {
```

Comment:
I think we should move all of this into one namespace: KVStore. No need for separate Action and AMOResponse namespaces, etc..